starfish/image/_learn_transform/translation.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from skimage.feature import register_translation
+from skimage.transform._geometric import SimilarityTransform
+
+from starfish.image._learn_transform.transforms_list import TransformsList
+from starfish.imagestack.imagestack import ImageStack
+from starfish.types import Axes, TransformType
+from starfish.util import click
+from ._base import LearnTransformBase
+
+
+class Translation(LearnTransformBase):
+
+    def __init__(self, reference_stack: ImageStack, axes: Axes, upsampling: int=1):
+        """
+        Parameters
+        ----------
+        axes:
+            The axes {r, ch, zplane} to iterate over
+        reference_stack: ImageStack
+            The target image used in skimage.feature.register_translation
+        upsampling: int
+            upsampling factor, defualt 1
+        """
+        self.upsampling = upsampling
+        self.axes = axes
+        self.reference_stack = reference_stack
+
+    def run(self, stack: ImageStack) -> TransformsList:
+        """
+        Iterate over the given axes of an ImageStack and learn the Similarity transform
+        based off the instantiated reference_image.
+
+        Parameters
+        ----------
+        stack : ImageStack
+            Stack to calculate the transforms on.
+
+        Returns
+        -------
+        List[Tuple[Mapping[Axes, int], SimilarityTransform]] :
+            A list of tuples containing axes of the Imagestack and associated
+            transform to apply.
+        """
+
+        transforms = TransformsList()
+        reference_image = np.squeeze(self.reference_stack.xarray)
+        for a in stack.axis_labels(self.axes):
+            target_image = np.squeeze(stack.sel({self.axes: a}).xarray)
+            if len(target_image.shape) != 2:
+                raise ValueError(
+                    "Only axes: " + self.axes.value + " can have a length > 1, "
+                                                      "please us the MaxProj filter."
+                )
+
+            shift, error, phasediff = register_translation(src_image=target_image,
+                                                           target_image=reference_image,
+                                                           upsample_factor=self.upsampling)
+            print(f"For {self.axes}: {a}, Shift: {shift}, Error: {error}")
```

Comment:
intended?  should this be gated on verbose?

Reply:
This was in the old fourier registration class so I kept it for continuity. @ambrosejcarr still needed?